js/desktopCapturer/getSources.js

```diff
@@ -52,6 +53,17 @@ function getSources(options, callback) {
         };
     }
 
+    if (!isMac) {
```

Comment:
I'd rather you check if this is Windows instead of not Mac. The below code will break if it's anything apart from Windows.